PhpcsChanged/Cli.php

```diff
@@ -128,7 +128,8 @@ function printHelp(): void {
 	printTwoColumns([
 		'--git-staged' => 'Compare the staged version to the HEAD version (this is the default).',
 		'--git-unstaged' => 'Compare the working copy version to the staged (or HEAD) version.',
-		'--git-branch <BRANCH>' => 'Compare the HEAD version to the HEAD of a different branch.',
+		'--git-branch <BRANCH>' => 'Compare the HEAD version to the HEAD of a different branch (deprecated).',
```

Comment:
What do you think of leaving `git-branch` as a supported alias rather than marking it as deprecated? I suspect that a lot of git developers know what a branch is but don't grok the idea that it's just a pointer to an "object". Of course, I guess there's some level of git knowledge required to understand `HEAD`. 🤔  

An alternative idea is just to document that branches are objects explicitly. Something like `--git-base-object <OBJECT>   Compare the HEAD version to the version found in OBJECT which can be a branch, commit, or other git object.`

Reply:
My logic was that aliases make it unnecessarily complicated. I updated the description with your text, and indicated what it was deprecated in favor of. I don't have strong feeling about this at this, and would be happy to just indicate it as a supported alias if you decide that is still better